pkg/registry/apis/identity/legacy/testdata/mysql--query_display-display_ids_uids.sql

```diff
@@ -1,9 +1,9 @@
 SELECT org_user.org_id, u.id, u.uid, u.login, u.email, u.name, 
   u.created, u.updated, u.is_service_account, u.is_disabled, u.is_admin
   FROM `user` as u JOIN org_user ON u.id = org_user.user_id
- WHERE org_user.org_id = ? AND ( 1=2
-   OR uid IN (?, ?)
-   OR u.id IN (?, ?)
+ WHERE org_user.org_id = 2 AND ( 1=2
+   OR uid IN ('a', 'b')
+   OR u.id IN (1, 2)
```

Comment:
For verifying the inputs got to the right places having the values inline is much nicer.  Obviously in real templates, this should be avoided.

Reply:
The whole point of the templating thing was that it was readable and that we are able to understand what we are actually writing and sending to the database, and that we have a test harness that provide us the right tools for it. I think this makes a great step forward in that direction, it looks really well.